module/zfs/vdev.c

```diff
@@ -3139,6 +3146,25 @@ vdev_dtl_reassess(vdev_t *vd, uint64_t txg, uint64_t scrub_txg,
 	mutex_exit(&vd->vdev_dtl_lock);
 }
 
+/*
+ * Iterate over all the vdevs except l2arc and spare and process kobj event
+ * depending on the request flag.
+ */
+void vdev_process_kobj_evt(vdev_t *vd, int request) {
```

Comment:
Please format properly. It complicates function search at lot.

Also, I first started thinking whether you really need two defines for the request, but then ended up thinking why not just create two separate functions to post and to clear? They would look pretty simple and compact, and without extra branching and argument.